src/main/java/com/google/jenkins/plugins/k8sengine/Manifests.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
Following up on #34, this should be changed to match the official header.

```suggestion
 * Copyright 2019 Google LLC
```